engine/dlib/src/dlib/buffer.cpp

```diff
@@ -56,6 +57,10 @@ namespace dmBuffer
         uint16_t m_Version;
         uint16_t m_ContentVersion;  // A running number, which user can use to signal content changes
         uint8_t  m_NumStreams;
+
+        // two point min/max definition of a AABB
+        dmVMath::Point3 boundsMin;
+        dmVMath::Point3 boundsMax;
```

Comment:
We should initialize them too, when the buffer is created.
I don't think they need certain constants, just initialize it to an "invalid" box.

Reply:
This seems to cancel my previous comment. With proper initialization we don't have something to worry about.